src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -235,6 +236,11 @@ private bool BuildInternal(string projectPath)
 
                         var root = _currentProject.ProjectDirectory;
 
+                        if(_currentProject.PackageFiles != null && _currentProject.PackageFiles.Any())
```

Comment:
add spaces after `if` and `foreach` instructions before you get too many comments on that (you have quite a few occurrences) also use `var`